Sources/AnyLint/Lint.swift

```diff
@@ -137,11 +137,36 @@
     ///   - customClosure: The custom logic to run which produces an array of `Violation` objects for any violations.
     public static func customCheck(checkInfo: CheckInfo, customClosure: (CheckInfo) -> [Violation]) {
         guard !Options.validateOnly else {
-            Statistics.shared.executedChecks.append(checkInfo)
+            Statistics.default.executedChecks.append(checkInfo)
             return
         }
 
-        Statistics.shared.found(violations: customClosure(checkInfo), in: checkInfo)
+        Statistics.default.found(violations: [checkInfo: customClosure(checkInfo)])
+    }
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)